### ISSUE TITLE
Rails/HttpPositionalArguments incorrectly fixes existing arguments

### ISSUE THREAD

**Issue by scytherswings** [bbatsov/rubocop#5896]
After updating to 0.56.0 my tests were broken due to this cop changing parameters in a few tests.
--------

## Expected behavior
The cop should not alter this line
```ruby
get v1_users_url, headers: { Authorization: token.token }
```

## Actual behavior
The cop inserted my headers into the `params` argument.
```ruby
get v1_users_url, params: { headers: { Authorization: token.token } }
```
## Steps to reproduce the problem

Run cop against a test with existing methods like so:
```ruby
require 'test_helper'

class UsersControllerTest < ActionDispatch::IntegrationTest
  test "Admin can get index" do
    token = authentications(:andrew_token)
    get v1_users_url, headers: { Authorization: token.token }

    assert_response :success
  end

  test "Tom can get his user info" do
    tom_token = authentications(:tom_token)
    get v1_user_url(users(:tom)), headers: { Authorization: tom_token.token }

    assert_response :success
  end
end
```

It will be changed to:
```ruby
require 'test_helper'

class UsersControllerTest < ActionDispatch::IntegrationTest
  test "Admin can get index" do
    token = authentications(:andrew_token)
    get v1_users_url, params: { headers: { Authorization: token.token } }

    assert_response :success
  end

  test "Tom can get his user info" do
    tom_token = authentications(:tom_token)
    get v1_user_url(users(:tom)), params: { headers: { Authorization: tom_token.token } }

    assert_response :success
  end
end
```

The source code from this example is in https://github.com/PinsterTeam/PinsterApi/tree/newell/bad_cop 
under `test/controllers/v1`

## RuboCop version

```
 PinsterApi git:(newell/bad_cop) rubocop -V
0.56.0 (using Parser 2.5.1.0, running on ruby 2.4.3 x86_64-linux)
```

**Comment by numbata** [comment]
@scytherswings fix already in master branch (see https://github.com/bbatsov/rubocop/pull/5893).
